PlottingScripts/Basic_Plot.py

```diff
@@ -0,0 +1,128 @@
+               x_axis_type=None, y_axis_type=None,
+               x_range=(-420, 420), y_range=(-420, 420),
+               min_border=0, outline_line_color="black",
+               background_fill="#f0e1d2", border_fill="#f0e1d2")
+
+    p.xgrid.grid_line_color = None
+    p.ygrid.grid_line_color = None
+
+    # annular wedges divided into smaller sections for bars
+    angles = np.pi/2 - big_angle/2 - df.index.to_series()*big_angle
+    # circular axes and labels
+    labels = np.power(10.0, np.arange(0, -5, -1))
+    labz = np.power(10.0, np.arange(0, -4.5, -0.5))
+    # Re-size if no values are above 0.1
+    if maxval < 0.1:
+        labels = np.delete(labels, 0)
+    # Size of radii of each label
+    pdb.set_trace()
+    rad = (((np.log10(labz / labels[0])) + labels.size) *
+           (outer_radius - inner_radius) / labels.size + inner_radius)
+    radii = (((np.log10(labels / labels[0])) + labels.size) *
+             (outer_radius - inner_radius) / labels.size + inner_radius)
+    # Add zero label to labels and radii
+    labels = np.append(labels, 0.0)
+    radii = np.append(radii, inner_radius)
+    rad = np.append(rad, inner_radius)
+    p.circle(0, 0, radius=rad, fill_color=None, line_color="white")
+    p.text(0, radii[:], [str(r) for r in labels[:]],
+           text_font_size="8pt", text_align="center", text_baseline="middle")
+    # Plot the values of each stat for each parameter.
+    for c in range(0, Cols.size):
+        p.annular_wedge(0, 0, inner_radius, (((np.log10(df[Cols[c]] /
+                                              labels[0])) + labels.size) *
+                                             (outer_radius - inner_radius) /
+                                             labels.size + inner_radius),
+                        -big_angle + angles + (2*c + 1)*small_angle,
+                        -big_angle + angles + (2*c + 2)*small_angle,
+                        color=stat_color[c])
+    # radial axes
+    p.annular_wedge(0, 0, inner_radius-10, outer_radius+10,
+                    -big_angle+angles, -big_angle+angles, color="black")
+
+    # Placement of parameter labels
+    xr = radii[0]*np.cos(np.array(-big_angle/2 + angles))
+    yr = radii[0]*np.sin(np.array(-big_angle/2 + angles))
+
+    label_angle = np.array(-big_angle/2+angles)
+    label_angle[label_angle < -np.pi/2] += np.pi
+    # Placing Labels and Legend
+    p.text(xr, yr, df.Parameter, angle=label_angle,
+           text_font_size="9pt", text_align="center", text_baseline="middle")
+
+    p.rect([-40, -40, -40], [18, 0, -18], width=30, height=13,
+           color=list(stat_color.values()))
+    p.text([-15, -15, -15], [18, 0, -18], text=Cols,
+           text_font_size="9pt", text_align="left", text_baseline="middle")
+
+    output_file("Sensitivity.html", title=Dataframe)
+    show(p)
+
+# Command call to implement example plot.
+makeplot('../../HDSAviz_data/analysis_CO.txt', np.array(['S1', 'ST']), np.array([0.001, 0.001]))
```

Comment:
Blake can you include this as comment in your code as usage??

sa['CO'][0] seems a bit wierd and confusing to use on it own.

can we instead put it as an intuitive named_variable ??

On Tue, Feb 23, 2016 at 4:09 PM, houghb <notifications@github.com> wrote:

> In PlottingScripts/Basic_Plot.py
> <https://github.com/houghb/HDSAviz/pull/2#discussion_r53873727>:
>
> > +    label_angle = np.array(-big_angle/2+angles)
> > +    label_angle[label_angle < -np.pi/2] += np.pi
> > +    # Placing Labels and Legend
> > +    p.text(xr, yr, df.Parameter, angle=label_angle,
> > +           text_font_size="9pt", text_align="center", text_baseline="middle")
> > +
> > +    p.rect([-40, -40, -40], [18, 0, -18], width=30, height=13,
> > +           color=list(stat_color.values()))
> > +    p.text([-15, -15, -15], [18, 0, -18], text=Cols,
> > +           text_font_size="9pt", text_align="left", text_baseline="middle")
> > +
> > +    output_file("Sensitivity.html", title=Dataframe)
> > +    show(p)
> > +
> > +# Command call to implement example plot.
> > +makeplot('../../HDSAviz_data/analysis_CO.txt', np.array(['S1', 'ST']), np.array([0.001, 0.001]))
>
> if you add the line import data_processing as dp in your import section,
> then this command to make an example plot will become:
>
> sa = dp.get_sa()
> makeplot(sa['CO'][0], np.array(['S1', 'ST']), np.array([0.001, 0.001]))
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/houghb/HDSAviz/pull/2/files#r53873727>.
>